package.json

```diff
@@ -14,6 +14,7 @@
     "compileProtos": "build/tools/compileProtos.js"
   },
   "dependencies": {
+    "@google-cloud/common": "^3.7.0",
```

Comment:
Is this only for the `Metadata` type?  We would prefer not to have an extra dependency just for that purpose. Would it be possible to copy the type?